packages/content/migrations.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Let's keep this is in `packages/content/migrations/migrations.json` for consistency with the other packages